src/lib/bm_ncp/ncp_uart.cpp

```diff
@@ -224,16 +225,15 @@ static bool bcmp_info_request_cb(uint64_t node_id) {
     }
   } else {
     // send back the info for the node_id
-    bcmp_request_info(node_id, &multicast_global_addr, bcmp_info_reply_cb, NULL);
```

Comment:
What's the story of removing the final parameter?

Reply:
This function always just had 3 parameters? Somehow we were able to pass 4 into it when it was a C++ function? Take a look at it from develop: https://github.com/bristlemouth/bm_protocol/blob/05b953254685c1463fc07c871c556f5f3cfad04f/src/lib/bcmp/bcmp_info.cpp#L33

Reply:
Cancel, that, somewhere in the lands between develop and where we are now this file somehow got an extra argument in these function calls? Let me look through bm_core history to see why.

Reply:
Looking at a diff of the history of that file in bm_core, I cannot see where an extra parameter was added to the info bcmp request. Maybe the NULL argument was added here by accident some time ago and this file was never built until now.

Reply:
yowsa, ok, thanks for cleaning it up

Reply:
Commit 4d77bb13 by me in bm_protocol added this in. Darn Git Blame.